examples/.railway/next-sse-chat/Dockerfile

```diff
@@ -20,11 +20,9 @@ COPY --from=build /app/out/full/ .
 COPY --from=build /app/out/pnpm-lock.yaml ./pnpm-lock.yaml
 
 # Some other stuff we need to build trpc packages
-COPY --from=build /app/tsconfig.json ./tsconfig.json
-COPY --from=build /app/tsconfig.build.json ./tsconfig.build.json
+COPY --from=build /app/tsconfig*.json ./
```

Comment:
Cursor is so good sometimes

I just wrote 

1. "fix this file according to https://github.com/trpc/trpc/pull/6271/files"
2. "make it a glob"